SoftLayer/managers/vs.py

```diff
@@ -1158,3 +1158,12 @@ def get_local_disks(self, instance_id):
         """
         mask = 'mask[diskImage]'
         return self.guest.getBlockDevices(mask=mask, id=instance_id)
+
+    def get_hardware_guests(self):
+        """Returns the hardware server vs associated.
+
+        :return SoftLayer_Hardware[].
+        """
+        object_filter = {"hardware": {"networkGatewayMemberFlag": {"operation": 0}}}
```

Comment:
`'{"hardware":{"virtualHost":{"id":{"operation":"not null"}}}}`  might be a better filter.

Reply:
Done.